Libraries/oneMKL/matrix_mul/README.md

```diff
@@ -0,0 +1,51 @@
+# matrix_mul_mkl sample
```

Comment:
Old styleReadme, please update using https://github.com/oneapi-src/oneAPI-samples/tree/master/DirectProgramming/DPC%2B%2B/CombinationalLogic/mandelbrot as a template

Reply:
Applied suggestions. Is it correct now?

Reply:
I don't see the needed changes. For example, there should be a purpose section